R/oauth-token.r

```diff
@@ -301,6 +301,7 @@ Token2.0 <- R6::R6Class("Token2.0", inherit = Token, list(
 #' @inheritParams oauth2.0_token
 #' @inheritParams jwt_signature
 #' @param secrets Secrets loaded from JSON file, downloaded from console.
+#' @param ... any additional claims for the token
```

Comment:
I think you can remove this line since we're already inheriting docs from `jwt_signature`?